modules/custom/openy_system/js/openy_system.js

```diff
@@ -0,0 +1,15 @@
+(function ($) {
+  "use strict";
+
+  /**
+   * Add confirmation message after uninstall package form submit.
+   */
+  Drupal.behaviors.openy_confirm_form = {
+    attach: function (context, settings) {
+      $("#packages-uninstall-submit").click(function () {
+        var msg = Drupal.t("Are you sure you want to uninstall packages?");
+        return confirm(msg)
```

Comment:
please move the message into the confirm function. No need to create a new variable